src/Tribe/Attendee_Repository.php

```diff
@@ -297,7 +297,7 @@ public function filter_by_optout( $optout ) {
 			case 'no_or_none':
 				$optout_keys = $this->attendee_optout_keys();
 				$optout_keys = array_map( [ $wpdb, '_real_escape' ], $optout_keys );
-				$optout_keys = '"' . implode( '", "', $optout_keys ) . '"';
+				$optout_keys = "'" . implode( '", "', $optout_keys ) . "'";
```

Comment:
Double quotes were still used here.

```suggestion
				$optout_keys = "'" . implode( "', '", $optout_keys ) . "'";
```